app/assets/stylesheets/custom.scss.css

```diff
@@ -0,0 +1,61 @@
+.pagination {
```

Comment:
please change the name of this file. `custom` doesn't mean anything. Can you imagine having a file with 1000+ lines? no thanks.

Create one file for the pagination, and then one for each  shared/page